src/Controller/Home.php

```diff
@@ -150,7 +156,7 @@ private function findPlaces($neLat, $neLon, $swLat, $swLon, $skipTracks, $locale
         /* @var $qResult \App\Entity\Place[] */
         $qResult = $qb
             ->select('g')
-            ->setMaxResults(10)
+            ->setMaxResults($size)
```

Comment:
Serving all places is not a good idea.
At the moment have 10s of places, but if we have 10000s of places we will run OOM on the server. Doctrine is very heavy ORM.
Data should be loaded on chunks.
The plugin works fine with dynamically loaded data.
In the future we may cache the places in static files or change the database.
Please revert to the default paging.